### DISCUSSION TITLE
希望Nginx-Cloudflare中，`redirect_url`支持Https

### DISCUSSION THREAD

**Discussion by despire1119** [MikeWang000000/Natter#71] [Ideas]
需求：
`Nginx-Cloudflare`的访问协议支持`HTTPS`的`redirect_url`跳转到`HTTP`的`direct_url`。

需求原因：
现代浏览器（Chrome、Edge等），通常会将`HTTP`请求自动转成`HTTPS`，这个转换几乎是强制的，比如最新版的Chrome无论关闭`HTTPS UPGRADE`还是`Delete domain security policies`它都不理你，所以希望我直接输入进浏览器的域名可以是`HTTPS`的。

需求延伸：
`direct_url`也为`HTTPS`。如果都为`HTTPS`连接的话我们就可以启用`Cloudflare`的`SSL`相关设置，提高安全性和访问响应速度。

**Comment by MikeWang000000** [comment]
支持啊，有个https选项 `cf_redirect_to_https`，设成True就行。
-p Web端口，需要指定为https端口

**Reply by despire1119** [reply]
好的，文档中写这里设置成false，我以为不能改，下班回去试试！感谢！
